js/ftx.js

```diff
@@ -1925,6 +1927,31 @@ module.exports = class ftx extends Exchange {
         return result;
     }
 
+    async cancelOrders (ids, symbol = undefined, params = {}) {
+        /**
+         * @method
+         * @name ftx#cancelOrders
+         * @description cancel multiple orders
+         * @param {[string]} ids order ids
+         * @param {string|undefined} symbol not used by ftx cancelOrders ()
+         * @param {object} params extra parameters specific to the ftx api endpoint
+         * @returns {object} a list of order ids queued for cancelation
+         */
+        await this.loadMarkets ();
+        const request = {
+            'orderIds': ids,
```

Comment:
FTX allows multiple cancelations using `clientIds`, so we should support it as well, check this example to see how we do it: 

https://github.com/ccxt/ccxt/blob/master/js/huobijp.js#L1489
https://docs.ftx.com/#bulk-cancel-orders-by-client-id